api/graphql/schema.graphql

```diff
@@ -257,6 +260,8 @@ type SiteInfo {
   periodicScanInterval: Int! @isAdmin
   "How many max concurrent scanner jobs that should run at once"
   concurrentWorkers: Int! @isAdmin
+  "The filter to use when generating thumbnails"
+  thumbnailMethod: Int! @isAdmin
```

Comment:
It would be nice to add a custom enum type for the valid thumbnail methods, like what has been done for the language preferences.

https://github.com/photoview/photoview/blob/446ef9a46485bd18bb1c6890d485344786a4e90c/api/graphql/schema.graphql#L273-L294